plasmapy/simulation/particle_integrators.py

```diff
@@ -0,0 +1,131 @@
+import math
+
+import numba
+import numpy as np
+from astropy import constants
+
+
+@numba.njit(parallel=True)
```

Comment:
Being a solar physicist (at least nominally), seeing `njit` always makes me think of [NJIT](https://www.njit.edu/), which has a really good solar physics grad program.  More relevantly, it's nice to see you making such good use of Numba!

Reply:
Oh, in New Jersey? I've heard everything's legal there.